js/views/home.vue

```diff
@@ -57,68 +50,53 @@ export default {
+            let userFollowersCount = this.metrics.metrics.followers || 0;
+            let orgFollowersCount = this.metrics.computed_metrics.followers_count || 0;
+            let isGoodAvailability = this.metrics.computed_metrics.resources_availability >= 80;
             return [{
-                value: this.$root.site.metrics.datasets || 0,
-                label: this._('Datasets'),
+                value: `${orgDatasetsCount} (${userDatasetsCount})`,
+                label: this._('Datasets (only yours)'),
                 icon: 'cubes',
                 color: 'aqua',
-                target: '#datasets-widget'
-            }, {
-                value: this.$root.site.metrics.reuses || 0,
-                label: this._('Reuses'),
-                icon: 'retweet',
-                color: 'green',
-                target: '#reuses-widget'
             }, {
-                value: this.$root.site.metrics.resources || 0,
-                label: this._('Resources'),
-                icon: 'file-text-o',
-                color: 'red',
-                target: '#resources-widget'
-            }];
-        },
-        communityBoxes: function() {
-            if (!this.$root.site.metrics) {
-                return [];
-            }
-            return [{
-                value: this.$root.site.metrics.users || 0,
-                label: this._('Users'),
-                icon: 'users',
-                color: 'yellow',
-                target: '#users-widget'
+                value: (this.metrics.computed_metrics.resources_availability || 0) + ' %',
+                label: this._('Availability of your latest datasets'),
+                icon: isGoodAvailability ? 'thumbs-up' : 'thumbs-down',
+                color: isGoodAvailability ? 'green' : 'red',
             }, {
-                value: this.$root.site.metrics.organizations || 0,
-                label: this._('Organizations'),
-                icon: 'building',
+                value: `${orgFollowersCount} (${userFollowersCount})`,
+                label: this._('Followers (only yours)'),
+                icon: 'heart',
                 color: 'purple',
-                target: '#organizations-widget'
             }];
         }
     },
     components: {
-        'chart': require('components/charts/widget.vue'),
-        'small-box': require('components/containers/small-box.vue')
-    },
-    watch: {
-        '$root.site.id': function(id) {
-            this.fetchMetrics();
-        }
+        sbox: require('components/containers/small-box.vue'),
+        reuses: require('components/reuse/list.vue'),
+        issues: require('components/issues/list.vue'),
+        discussions: require('components/discussions/list.vue'),
     },
     attached: function() {
-        this.fetchMetrics();
+        this.update();
+        this.$root.me.$on('updated', this.update.bind(this));
+    },
+    detached: function() {
+        this.update();
+        this.$root.me.$on('updated', null);
```

Comment:
Should be:
```javascript
this.$root.me.$off('updated', handler)
```
where handler is the attached handler.
Given the ``bind(this)`` in ``attached``, I don't know if it will match the correct handler.
Try this:
```javascript
attached: function() {
     this._hander = this.$root.me.$on('updated', this.update.bind(this));
},
deteched: function() {
    this._handler.remove();
}
```